### PR TITLE
Connect prophet jenkinsfile trello#3681

### PR DESCRIPTION
**Pull request by digitaltom**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@kirushik will review your pull request :dancers: check https://trello.com/c/o9vZTS39/3681-migrate-from-river-suse-de-to-ci-suse-de
  @kirushik: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/8/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged fe7295e3672d3b9c9726cdc5cdd86de3b2602d31 into 5c7a8ab16b940c9eb236961edbdbe25cf06c291a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/SCC-Connect-Prophet/191911/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged fe7295e3672d3b9c9726cdc5cdd86de3b2602d31 into 5c7a8ab16b940c9eb236961edbdbe25cf06c291a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/9/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 63725490cd9307b4c5982f8a310b2cfe96fcb11f into 5c7a8ab16b940c9eb236961edbdbe25cf06c291a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/17/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 63725490cd9307b4c5982f8a310b2cfe96fcb11f into 66c17e89a3848e42be8db6bc8291e727fae0fa8a )

File: Jenkinsfile-Prophet [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  stage('checkout') {
+     git 'git@github.com:SUSE/connect.git'
+  }
+
+  ansiColor('xterm') {
+
+    stage('run prophet')
+    {
+      sh 'cd prophet; ruby ./prophet.rb'
```
Comment (kirushik) [inline Jenkinsfile-Prophet]: I think this should be done like 
```groovy
dir('./prophet') {
  sh 'ruby ./prophet.rb'
}
```

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/1865/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 3eff5154413474a869792d03cb234735ed60920b into 66c17e89a3848e42be8db6bc8291e727fae0fa8a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/1866/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 99bf287ad5b563f7b86ca5b37a2ac2ab6c6f739d into 66c17e89a3848e42be8db6bc8291e727fae0fa8a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/1867/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 95412d76c0be0000625f7f0c69bf731e5d692115 into b91762d8262fec144b7bf504c86b0982bc9fc85e )

**Pull request merged**